source/extensions/common/wasm/wasm.h

```diff
@@ -189,8 +189,9 @@ class Context : public Http::StreamFilter,
   //
   // VM level downcalls into the WASM code on Context(id == 0).
   //
+  virtual bool onValidateConfiguration(absl::string_view configuration);
   virtual void onStart(absl::string_view root_id, absl::string_view vm_configuration);
-  virtual void onConfigure(absl::string_view configuration);
+  virtual bool onConfigure(absl::string_view configuration);
```

Comment:
Shouldn't this be `WasmResult`?

Reply:
These are calls into the VM.  WasmResult is for calls coming out of the VM current (environment error codes).  Do we want to conflate the two?  Do we want a different enum or is a bool OK?

Reply:
Good point.